beginner_source/export_simple_model_to_onnx_tutorial.py

```diff
@@ -0,0 +1,150 @@
+# -*- coding: utf-8 -*-
+"""
+Export a PyTorch model to ONNX
+==============================
+
+**Author**: `Thiago Crepaldi <https://github.com/thiagocrepaldi>`_
+
+.. Note::
+    As of PyTorch 2.1, there are two versions of ONNX Exporter.
+
+    * ``torch.onnx.dynamo_export` is the latest and recommended exporter basedon the TorchDynamo and is the default starting from PyTorch 2.1
+    * ``torch.onnx.export`` is based on TorchScript backend and has been the default until PyTorch 2.0.
+
+In this tutorial, we describe how to convert a model defined in PyTorch into the ONNX format using
+the latest and preferred ``torch.onnx.dynamo_export` ONNX exporter.
+
+"""
+
+###############################################################################
+# In the `60 Minute Blitz <https://pytorch.org/tutorials/beginner/deep_learning_60min_blitz.html>`_,
+# we had the opportunity to learn about PyTorch at a high level and train a small neural network to classify images.
+#
+# While PyTorch is great for iterating on the development of models, the resulting models are not typically deployed
```

Comment:
I'd refrain from making comments about pytorch like " the resulting models are not typically deployed to production in this fashion. This is where `ONNX <https://onnx.ai/>`_ (Open Neural Network Exchange) comes in!"

Can just remove this line and meaning will still be intact